src/scripts/events/domEvents.js

```diff
@@ -5,41 +5,39 @@ import { deleteAuthorBooks, viewBookDetails } from '../../api/mergedData';
 import { showAuthors } from '../components/pages/authors';
 import addBookForm from '../components/forms/addBookForm';
 import addAuthorForm from '../components/forms/addAuthorForm';
-import { updateAuthor } from '../../api/authorData';
+import { getSingleAuthor } from '../../api/authorData';
 // import selectAuthor from '../components/forms/selectAuthor';
 import viewBook from '../components/pages/viewBook';
 // import { showAuthors } from '../components/pages/authors';
 
-const domEvents = () => {
+const domEvents = (uid) => {
   document.querySelector('#main-container').addEventListener('click', (e) => {
     // TODO: CLICK EVENT FOR DELETING A BOOK
     if (e.target.id.includes('delete-book')) {
       // eslint-disable-next-line no-alert
       if (window.confirm('Want to delete?')) {
         const [, firebaseKey] = e.target.id.split('--');
 
-        deleteBook(firebaseKey).then((booksArray) => showBooks(booksArray));
+        deleteBook(firebaseKey, uid).then((booksArray) => showBooks(booksArray));
       }
     }
 
     // TODO: CLICK EVENT FOR SHOWING FORM FOR ADDING A BOOK
     if (e.target.id.includes('add-book-btn')) {
-      addBookForm();
+      addBookForm({}, uid);
```

Comment:
```suggestion
addBookForm(uid)
```
1: addBookForm has uid as it first parameter so that must be the first argument you pass 
2: you do need to pass an empty object when calling the function. addBookForm obj has a set default value of an empty object.